packages/python/src/components/TypeExpression.tsx

```diff
@@ -0,0 +1,71 @@
+import { Children, For, memo } from "@alloy-js/core";
+
+export interface SingleTypeExpressionProps {
+  children: Children;
+}
+
+export function SingleTypeExpression(props: SingleTypeExpressionProps) {
+  const resolvedChildren = memo(() => props.children);
+
+  return (
+    <group>
+      <indent>
+        <sbr />
+        {resolvedChildren}
+      </indent>
+      <sbr />
+    </group>
+  );
+}
+
+export interface UnionTypeExpressionProps {
+  children: SingleTypeExpressionProps[];
```

Comment:
Is it a convention to call these fields `children`? If so that's fine, but if not it seems like we could use more descriptive names.

Reply:
I don't mean `Children`, I mean `children` 🙂 
As in, not the `Children` type but the use of the name "children" for a property.